src/de/jost_net/JVerein/io/FormularAufbereitung.java

```diff
@@ -721,18 +721,14 @@ public void addZUGFeRD(Rechnung re, boolean mahnung) throws IOException
     for (SollbuchungPosition sp : re.getMitgliedskonto()
         .getSollbuchungPositionList())
     {
-      Double betrag = sp.getNettobetrag();
-      if (betrag == null || betrag == 0)
-      {
-        betrag = sp.getBetrag();
-      }
+      BigDecimal betrag = sp.getNettobetrag();
       if (sp.getBetrag() < 0)
       {
         invoice.addItem(new Item(
             new Product(sp.getZweck(), "", "LS",
                 new BigDecimal(sp.getSteuersatz()).setScale(2,
                     RoundingMode.HALF_DOWN)),
```

Comment:
Das müsste eigentlich `RoundingMode.HALF_UP` sein. Da hoffentlich in `SollbuchungPositionImpl` gerundet wird (siehe weiter unten), brauchen wir hier gar nicht mehr zu runden und die ganze Arithmetik bleibt zusammen und ohne Code Duplications.